### DISCUSSION TITLE
Specifying the fields of a list collection

### DISCUSSION THREAD

**Discussion by peterhijma** [pocketbase/pocketbase#2925] [Q&A]
I tried this:

```js
await pb.collection("foo").getList(null, null, { fields: "title, id" });
```

It's a bit unclear to me why the response does contain other (built-into-pocketbase?) fields as well, but as an empty string:

```js
[ { "id": "01czlxyjthmxopl", "title": "Foo", "created": "", "updated": "", "collectionId": "", "collectionName": "", "expand": {} },... ]
```

Is there a reason for it? Just trying to understand :)

**Comment by ganigeorgiev** [comment]
Those fields are available always as part of the [Record wrapper class](https://github.com/pocketbase/js-sdk/blob/master/src/models/Record.ts).

If you inspect the browser's network request they shouldn't exist in the raw json response.

**Comment by peterhijma** [comment]
> Those fields are available always as part of the [Record wrapper class](https://github.com/pocketbase/js-sdk/blob/master/src/models/Record.ts).
> 
> If you inspect the browser's network request they shouldn't exist in the raw json response.

I see, thanks, for the extremely quick response, as always 🙏
